core/init.php

```diff
@@ -5,6 +5,7 @@
 
     global $pdo;
     session_start();
+    $_SESSION["user"] = "NULL";

```

Comment:
It doesn't need to be set to NULL. If it is set, then a session is created/started. We can use `isset()` to check whether a value is set for the 'user' key.